config/routes

```diff
@@ -9,3 +9,6 @@
 /comments CommentR POST
 
 /profile ProfileR GET
+
+-- By default this file is used by `parseRoutesFile` in Foundation.hs
```

Comment:
Why put this at the bottom, but the other comment at the top?

Reply:
Ah that's my bad. I wanted them both at the top since I thought that's the first place users would look + it would stay visible as the file grew large. I just messed up moving them from my own project to the scaffolding

Reply:
Fixed, nice catch 👍